test/views/pages/confirmation/confirmFPSpec.scala

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.pages.confirmation
+
+import org.jsoup.Jsoup
+
+import play.api.i18n.Messages.Implicits._
+import testHelpers.ViewSpecHelpers.confirmation.confirmFP
+import testHelpers.ViewSpecHelpers.CommonViewSpecHelper
+import testHelpers.ViewSpecHelpers.CommonMessages
+import views.html.pages.confirmation.{confirmFP => views}
+
+class confirmFPSpec extends CommonViewSpecHelper with confirmFP with CommonMessages {
```

Comment:
Classes must also have capitalised first letters. Also CommonMessages is imported by the ViewSpecHelper and is not needed as an extension here.